<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paradialog_bigimagewarp</class>
 <widget class="QDialog" name="Paradialog_bigimagewarp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BigImageWarp v1.0</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_fileio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>691</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>File IO: </string>
   </property>
   <widget class="QLabel" name="label_img_sub_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input subject image format and folder(to be warpped): </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input subject marker file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_sub">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_sub">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for dir...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_marker_sub">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for file...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marker_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marker_tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_sub2tar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input target marker file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_marker_tar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_img_warp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse for dir...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_swc_grid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output warped image folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_warp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Vaa3D raw（series，2D）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vaa3D raw</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_paras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Paras:</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Device mode:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_device_cpu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_device_gpu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string> Program run mode:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_affine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Affine</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_stps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Non-rigid</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>groupBox_paras</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_fileio</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Paradialog_bigimagewarp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Paradialog_bigimagewarp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
